### PR TITLE
Just added a few elements

### PR DESCRIPTION
**Pull request by dann3wman**
2019-09-18 15_58_39-faces js editor - Brave
2019-09-18 15_57_48-faces js editor - Brave
2019-09-18 15_59_00-faces js editor - Brave
2019-09-18 16_27_54-faces js editor - Brave

Just added a few new face elements and some new transform parameters so eyebrows can be rotated, hair can be recolored, mouths can change size, etc. Going to do some more fiddling incrementally.

### CODE REVIEW

**Comment by dumbmatter** [comment]
I'm really excited to see this PR! This is exactly what I wanted to happen, SVGs you drew in Inkscape getting integrated right in!

You mention maybe doing a "complete revamp" on Discord and I'm sure that could have a better end result. But either way, I'm going to leave you a few generic comments :)

First -  it's easier if you don't work directly on the master branch when you forked someone's repo. Instead, make some other branch to work in. That way, you can keep your master branch in sync with mine in the future. Like if I make another commit to my master branch before accepting your PR, then it'll be a little messy for you to sync them up (not impossible, just a little annoying).

File: package.json [hunk 1]
```diff
@@ -52,6 +52,8 @@
     }
   },
   "dependencies": {
-    "ws": "^7.1.2"
+    "facejs": "^1.0.1",
```
Comment (dumbmatter) [inline package.json]: This file (and yarn.lock) shouldn't be changed.

**Comment by dumbmatter** [comment]
I need to play around with it some to have more feedback. I'll try to do that tomorrow, or maybe this weekend.

Thanks again, this is really cool!

File: package.json (hunk 1 above)
Comment (dann3wman) [reply package.json]: I think my IDE did that automatically. I just barely even know what those files even do. I'll keep them protected going forward.

**Comment by dann3wman** [comment]
Thanks a bunch for providing me many hours of free entertainment! Glad to be able to contribute something back to the project. I am pretty new to collaborating through Git & coding in JavaScript so any feedback, no matter how basic, is very welcome.

**Comment by dann3wman** [comment]
drawing_player

What do you think of this style? I was thinking that each team could have 2 color values for the jersey so the player image could have a corresponding jersey color to what team they are on. Let me know your thoughts.

**Comment by dumbmatter** [comment]
it's better than what i have now! but it's also not as good as what someone else is working on... https://github.com/TravisJB89/facesjs/tree/smile

image

**Pull request closed without merging**